topiary-queries/queries/tree_sitter_query.scm

```diff
@@ -53,7 +54,7 @@
 ; zero-byte nodes, so we just need to set the spacing correctly.
 (predicate
   (parameters
-    _ @prepend_space
+    _ @prepend_spaced_softline
   )
 )
```

Comment:
This is nice, but I think it would be worthwhile also adding indentation here as well. So, we get something like:

```scm
  (#any-of? @function.builtin
    "union"
    "difference"
    "intersection"
    "circle"
  )
```

IMO, having the capture name on the same line as the predicate "looks nice", but that's not essential.

Reply:
From the predicates section of the [book](https://tree-sitter.github.io/tree-sitter/using-parsers/queries/3-predicates-and-directives.html):

> Predicate S-expressions start with a _predicate name_ beginning with a `#` character, and ending with a `?` character. After that, they can contain an arbitrary number of `@`-prefixed capture names or strings

I think for now I will go with the assumption that:
*  there is a readable limit to predicate tags and identifiers (such as ` (#set! injection.language "doxygen")`)
* all indentifiers and capture tags precede literals

Reply:
I've implemented the suggestion with this output:
https://github.com/tweag/topiary/blob/2dea7e7e154bef420f6e8548b25fe5a6c4586e83/topiary-cli/tests/samples/expected/tree_sitter_query.scm#L1516-L1526